platform/security/log-sanitizer/src/main/java/org/codice/ddf/log/sanitizer/LogSanitizer.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.log.sanitizer;
+
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.commons.lang.StringUtils;
+
+public class LogSanitizer {
+
+  private final Object object;
+
+  public LogSanitizer(Object object) {
+    this.object = object;
+  }
+
+  public static LogSanitizer cleanAndEncode(Object object) {
```

Comment:
✏️ `sanitize` (since the name of the class is `LogSanitizer`)

Or you could even call it something like `prepareForLogging` or `wrapWithSanitizer` since the sanitization is actually done elsewhere.